plugins/mcreator-localization/lang/texts.properties

```diff
@@ -766,6 +766,7 @@ blockly.category.commands=Command parameters
 blockly.category.return=Return blocks
 blockly.category.blockpredicates=Conditional placements
 blockly.category.features=Features
+blockly.category.intproviders=Integer providers
```

Comment:
You should maybe change Integer for something else like "Number providers". Most of the users don't know the coding words and we never use `Integer` in the UI, so it might confuse users.

Reply:
We use the word `number` for double dependencies, also you use it for entity data feature, don't you?
I'd say it's fine as it indicates numbers are not real but integer here.

Reply:
> We use the word `number` for double dependencies, also you use it for entity data feature, don't you? I'd say it's fine as it indicates numbers are not real but integer here.

I also used it for integers in some places (such as command arguments). I'm using Number for entity data ([here](https://github.com/MCreator/MCreator/pull/2007/files#diff-a7476eb41eb680120b4ec6fe94b52a2196fabb286737fef298b5d10286fae1c0R40))

Reply:
This is to differentiate them from Float providers, which are used by the dripstone features. The blockstate procedure blocks also use "integer" in their text

Reply:
When possible we try to merge int/long and float/double into a single type called number.

But I think features can't accept int config in a place where float config would be accepted and vice versa (providers are objects and can't be cast one into another). This is probably why the decision to use int and float was used, although this may confuse the users.